kafka-3.8.yaml

```diff
@@ -73,6 +73,8 @@ subpackages:
           ln -s /usr/lib/kafka/libs ${{targets.subpkgdir}}/opt/bitnami/kafka/libs
           ln -s /usr/lib/kafka/config ${{targets.subpkgdir}}/opt/bitnami/kafka/config
           ln -s /usr/lib/kafka/logs ${{targets.subpkgdir}}/opt/bitnami/kafka/logs
+          mv ${{targets.contextdir}}/usr/lib/kafka/config/server.properties ${{targets.contextdir}}/usr/lib/kafka/config/server.properties.original
```

Comment:
a quick fs lookup on `bitnami/kafka:3.8` image shows me that the file is located at some other place.

```bash
 $ fd properties.original
opt/bitnami/kafka/config/server.properties.original
```
Is there a reason to put it inside `/usr/lib` here? 

I think we should proceed with the following: 

```bash
cp /usr/lib/kafka/config/server.properties ${{targets.contextdir}}/opt/bitnami/kafka/config/server.properties.original
```

Reply:
I'm also not certain that contextdir works in that situation 🤔 so it could be targets.destdir to targets.subpkgdir instead of contextdir

Reply:
I think given it's under `subpackages` I think contextdir should figure out that given it's a subpackage it should be `subpkgdir` 

Looking at other bitnami/compat usage, I think we are good with contextdir as well but we can move to subpkgdir for sure to be more specific. 

Also, I think given you're copying this file `usr/lib/kafka/config/server.properties` from main package pipeline to subpackage one, you can either use `cp /usr/lib/kafka/config/server.properties ${{targets.contextdir}}/opt/bitnami/kafka/config/server.properties.original` or `cp ${{targets.destdir}}/usr/lib/kafka/config/server.properties ${{targets.contextdir}}/opt/bitnami/kafka/config/server.properties.original` 

@developer-guy please let me know if I missed something here.

Reply:
Locally I've tried:
```
# encounters ...WARN mv: can't rename '/home/build/melange-out/kafka-3.8/usr/lib/kafka/config/server.properties': No such file or directory
mv ${{targets.destdir}}/usr/lib/kafka/config/server.properties ${{targets.contextdir}}/usr/lib/kafka/config/server.properties.original
mv ${{targets.destdir}}/usr/lib/kafka/config/server.properties ${{targets.subpkgdir}}/usr/lib/kafka/config/server.properties.original

# encounters ...WARN mv: can't rename '/usr/lib/kafka/config/server.properties': No such file or directory
mv /usr/lib/kafka/config/server.properties ${{targets.subpkgdir}}/usr/lib/kafka/config/server.properties.original
```

So far can only get the following to build successfully:
```
mv ${{targets.destdir}}/usr/lib/kafka/config/server.properties ${{targets.destdir}}/usr/lib/kafka/config/server.properties.original
```